docs/src/reference/workflow-creation.md

```diff
@@ -0,0 +1,89 @@
+# Workflow Instance Creation
+
+Depending on the workflow definition, an instance of it can be created in various ways.
+
+## Explicit creation of a workflow instance
+
+A workflow instance can be created by sending a command specifying the process id or the unique key of the workflow.
+There are two commands to create a workflow instance.
+
+### Asynchronous execution
```

Comment:
What do you think about aligning the two headlines? 
* `Asynchronous Execution` vs. `Synchronous Execution` 
* `Create and Execute Asynchronously` vs. `Create and Await Results`